Packs/UrlScan/Integrations/UrlScan/UrlScan.py

```diff
@@ -384,7 +384,7 @@ def format_results(client, uuid):
             dbot_score['Indicator'] = demisto.args().get('url')
             dbot_score['Score'] = 0
             dbot_score['Type'] = 'url'
-            human_readable['Malicious'] = 'Benign'
+            human_readable['Malicious'] = 'Unknown'
```

Comment:
i think that the header name Malicious is wrong here.
should be completely removed in case the score is `0`.
or be called `reputation` and have 1 of the values: (Unknown, Benign, Suspicious, Malicious)